pangeo/values.yaml

```diff
@@ -27,6 +27,15 @@ jupyterhub:
   prePuller:
     hook:
       enabled: false
+  # optimizations: see https://zero-to-jupyterhub.readthedocs.io/en/latest/optimization.html?highlight=scheduling#optimizations
+  scheduling:
+    userScheduler:
+      enabled: true
+    podPriority:
+      enabled: true
+    userPlaceholder:
```

Comment:
What are the added podPriority and userPlaceholder in addition to userScheduler?

Reply:
Pod priotiry allows you to set some things as more important than others. Partialularly useful to let new users log in to an over subscribed cluster as low priority things (like some user's 100th dask worker) can be kicked off in exchange for the new users notebook.

The placeholder adds some low priority dummy pods which are there to save a space for new users to log into. This means a cluster scale will be triggered before the cluster is full instead of at the moment of realizing it's full.